src/commands/detail.js

```diff
@@ -114,6 +114,14 @@ module.exports = async function detail(flags) {
 		repositories.sort((a, b) => new Date(b.updatedAt) - new Date(a.updatedAt));
 	}
 
+	if (flags.names) {
+		const repos = [];
+		repositories.forEach((repository) => {
+			repos.push(repository.nameWithOwner);
+		});
+		console.log(repos);
```

Comment:
```suggestion
		console.log(repositories.map(({ nameWithOwner }) => nameWithOwner));
		return;
```

Reply:
altho, this currently runs into the console log limit - i get:
```
  'ljharb/call-bind',
  'ljharb/define-properties',
  'inspect-js/is-generator-function',
  ... 139 more items
]
```
i think maybe we might need:
```suggestion
		repositories.forEach((repository) => {
			console.log(repository.nameWithOwner);
		});
		return;
```

Reply:
Confiriming, so which do you suggest using afterall, the one-line `console.log(repositories.map(({ nameWithOwner }) => nameWithOwner));`

or the latter code block?

Reply:
I think the latter one, so that each name is printed on its own line, as opposed to being a logged array (with the brackets and colors and everything)

Reply:
Aight. That is what I've used in the latest commit yep